calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,227 @@
+---
+description: Configure Calico Enterprise for NodeLocal DNSCache
+---
+
+# Configure Calico Enterprise for NodeLocal DNSCache
```

Comment:
Title should reflect what user wants to do: improve performance. The method is to configure NodeLocal DNSCache, which is revealed in the big picture.

Improve cluster DNS performance

Reply:
done